bee-tangle/src/unconfirmed_message.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_common::packable::{Packable, Read, Write};
+use bee_message::MessageId;
+
+use std::ops::Deref;
+
+/// A type representing an unconfirmed message.
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub struct UnconfirmedMessage(MessageId);
```

Comment:
Sorry for only noticing that now but could you rename this to `UnreferencedMessage` please ? It was renamed on some other places and it would be more consistent. Also the module name :)

Reply:
Sure